multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/sql/delete_unlink_file_content_triggers.sql

```diff
@@ -0,0 +1,5 @@
+DROP TRIGGER IF EXISTS DELETE_FROM_SYSTEM_TABLE_LM_SL_PERSISTANCE_FILE ON LM_SL_PERSISTENCE_FILE;
```

Comment:
I understand that this file is called in case of a rollback but wouldn't it be cleaner if these DROP statemets are executed immediately prior to the trigger's creation? 
Or is it neccessary to have them deleted in a rollback?